packages/csv-parser-state/test/main.spec.js

```diff
@@ -125,12 +125,14 @@ describe('CsvParserState', () => {
 
   describe('::_transformTransitions', () => {
     describe('without state transitions', () => {
-      test('should not mutate state', async () => {
+      test('should not mutate state', () => {
         const actual = {
           foo: 'bar',
           key: 'my-state',
         }
-        expect(csvParser._transformTransitions(actual)).resolves.toEqual(actual)
+        return expect(csvParser._transformTransitions(actual)).resolves.toEqual(
```

Comment:
There were a bunch of tests that did not return the expectation with the promise.